das_api/src/api/api_impl.rs

```diff
@@ -150,7 +150,7 @@ impl ApiContract for DasApi {
         let owner_address = validate_pubkey(owner_address.clone())?;
         let owner_address_bytes = owner_address.to_bytes().to_vec();
         let sort_by = sort_by.unwrap_or_default();
-        self.validate_pagination(&limit, &page, &before, &after)?;
+        self.validate_pagination(&limit, &mut page, &before, &after)?;
```

Comment:
Instead of making page mutable everywhere (when it doesnt need to be), do an `page.unwrap_or(1)` instead before the validation.

Reply:
> Instead of making page mutable everywhere (when it doesnt need to be), do an `page.unwrap_or(1)` instead before the validation.

If we default page to 1 before the validation, then [this](https://github.com/helius-labs/digital-asset-rpc-infrastructure/pull/66/files#diff-9dc700dc1d7d823131815f661e774ce3f6ef26d1a0b0da393a56209f73395f13R77) will return an error if the user provides either `before` or `after` but not `page` (even though it shouldn't).

Reply:
Yeah I guess it works. But generally mutating a value within a validation function is bad practice.

Reply:
What'd be a better approach for this?